tests/assets/SPECS/rpm-file-attrs.spec

```diff
@@ -0,0 +1,68 @@
+Name:           rpm-file-attrs
+Version:        1.0
+Release:        1
+Summary:        Test RPM file attributes
+License:        MIT
+BuildArch:	    noarch
+
+%description
+%{summary}
+
+%install
+mkdir -p %{buildroot}/opt/%{name}
+
+echo "normal" > %{buildroot}/opt/%{name}/normal
+mkdir -p %{buildroot}/opt/%{name}/dir/
+
+# symlinks to a file and directory
+touch %{buildroot}/opt/%{name}/symlink
+mkdir -p %{buildroot}/opt/%{name}/symlink_dir/
+ln -sf ../normal %{buildroot}/opt/%{name}/symlink
+ln -sf ../dir/ %{buildroot}/opt/%{name}/symlink_dir/
```

Comment:
This is also really confusing.  The only way I was able to create a symlink and get the package to build without a warning, was to touch the files first, and then create the link with overwriting enabled.  

I strongly suspect I'm doing something wrong here.